avocado/core/messages.py

```diff
@@ -413,3 +415,27 @@ def handle(self, message, task, job):
             os.makedirs(os.path.dirname(file), exist_ok=True)
         self._save_message_to_file(filename, message['log'], task,
                                    message.get('encoding', None))
+
+
+class OutputMessageHandler(BaseRunningMessageHandler):
+    """
+    Handler for displaying messages in UI.
+    It will show the message content in avocado UI.
+    :param status: 'running'
+    :param type: 'output'
+    :param log: output message
+    :type log: bytes
+    :param encoding: optional value for decoding messages
+    :type encoding: str
+    :param time: Time stamp of the message
+    :type time: float
+    example: {'status': 'running', 'type': 'output',
+             'log': 'this is the output', 'time': 18405.55351474}
+    """
```

Comment:
nitpick: could you please add the extra lines between the descriptions, parameters and example?